Source/Core/DiscIO/VolumeGC.cpp

```diff
@@ -193,8 +193,14 @@ void CVolumeGC::LoadBannerFile() const
 
   GCBanner banner_file;
   std::unique_ptr<IFileSystem> file_system(CreateFileSystem(this));
-  size_t file_size = (size_t)file_system->GetFileSize("opening.bnr");
+  if (!file_system)
+    return;
+
+  std::unique_ptr<IFileInfo> file_info = file_system->FindFileInfo("opening.bnr");
+  if (!file_info)
+    return;
 
+  size_t file_size = (size_t)file_info->GetSize();
```

Comment:
This was necessary in the past because GetSize used to return u64.

In practice, I guess we won't run into any system where `sizeof(size_t) < sizeof(u32)`, but the C++ standard doesn't guarantee `sizeof(size_t) >= sizeof(u32)` as far as I know. @lioncash, should the cast be removed or not? (If not, I'll make it a `static_cast`.)